src/rules/no-redundant-flow.ts

```diff
@@ -1,48 +1,109 @@
-import { contextUtils, createRule } from "../utils";
+import {
+    AST_NODE_TYPES,
+    TSESTree,
+} from "@typescript-eslint/experimental-utils";
+import { pipe } from "fp-ts/function";
+import * as NonEmptyArray from "fp-ts/NonEmptyArray";
+import * as O from "fp-ts/Option";
+import { contextUtils, createRule, prettyPrint } from "../utils";
+
+const getArgumentExpression = (
+    x: TSESTree.CallExpressionArgument
+): O.Option<TSESTree.Expression> =>
+    // TODO: isExpression?
```

Comment:
I was hoping an existing predicate would exist for this but I couldn't find one.